DMTR-231.tex

```diff
@@ -23,27 +23,27 @@
 
 \begin{document}
 
-\def\milestoneName{ComCam OPS Readiness}
-\def\milestoneId{LDM-503-11a}
+\def\milestoneName{All P1a DM requirements verified.}
+\def\milestoneId{LDM-503-19a}
+\def\testplanId{LVV-P72}
 \def\product{Data Management}
 
 \setDocCompact{true}
 
-\title{LDM-503-11a: ComCam OPS Readiness Test Plan and Report}
+\title{\testplanId: \milestoneId\,\textendash\,\milestoneName }
 \setDocRef{\lsstDocType-\lsstDocNum}
-\date{\vcsdate}
-\author{ Robert Gruendl }
+\date{\date}
+\author{Leanne Guy}
 
 \input{history_and_info.tex}
 
 
 \setDocAbstract{
-This is the test plan and report for
-\textbf{ ComCam OPS Readiness} (LDM-503-11a),
+This is the test plan an report for \textbf{All P1a DM requirements verified}\,(LDM-503-19a),
```

Comment:
an --> and

Reply:
argh - did not fix the link, only the text